busmon.py

```
@@ -357,49 +355,69 @@ async def websocket_endpoint(websocket: WebSocket):
+    daemon-thread + `asyncio.run_coroutine_threadsafe` bridge is gone:
+    handlers schedule broadcasts directly on the running loop.
+    """
+    bus = AsyncMessageBusClient()
+
+    def _on_raw(raw: str) -> None:
+        # Handler dispatch on the async client is synchronous (matches the
+        # pyee EventEmitter contract), but we are already on the asyncio
+        # loop — schedule the awaitable broadcast directly.
+        try:
+            payload = _build_message_payload(raw)
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)